browser/brave_wallet/sources.gni

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+brave_browser_brave_wallet_sources = [
```

Comment:
so based on dm discussion brave_wallet_tab_helper shouldn't need to be in sources.gni anymore so I would move it back to BUILD.gn and then create a subdir for notifications so we don't have sources.gni and BUILD.gn in the same directory

Reply:
done